src/app/workflow/launch-third-party/launch-third-party.component.html

```diff
@@ -82,6 +94,24 @@ <h3>Launch with</h3>
               ><mat-icon class="anvil-icon" svgIcon="anvil"></mat-icon> AnVIL &raquo;</a
             >
           </div>
+          <div fxFlex="100%" *ngIf="workflow?.descriptorType === 'WDL'">
+            <a
+              mat-raised-button
+              [matTooltip]="
+                !(hasContent$ | async)
```

Comment:
expecting `[matTooltip]="blah"` similar to the other button

Reply:
Updated